flow/check.js

```diff
@@ -1,9 +1,4 @@
 const cp = require('child_process')
 const flow = require('flow-bin')
 
-try {
```

Comment:
guess is that exit codes are not making it out of the cli all together, see above.  will need to try some other way of getting exit 1 bubbled.